packages/integration-tests/projects/suite-desktop/tests/first.spec.js

```diff
@@ -0,0 +1,61 @@
+const { _electron: electron } = require('playwright');
+const { test, expect } = require('@playwright/test');
+const { Controller } = require('../../../websocket-client');
+
+let window, electronApp;
```

Comment:
I would declare it into 2 likes like below, as a good practice and better readability.
```suggestion
let window;
let electronApp;
```